_posts/2016-04-27-introducing-infractores.markdown

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Introducing Infractores"
+date: 2016-04-28 15:37:00
+categories: ["open-source", "crowdsourcing"]
+author: "etagwerker"
+---
+
+I've always been a big fan of [scratching your own itch](https://en.wikipedia.org/wiki/The_Cathedral_and_the_Bazaar#Lessons_for_creating_good_open_source_software). My latest itch was the **insane amount of parking
+violations** that I see everyday in **Buenos Aires**, near [our office](https://foursquare.com/v/ombulabs-hq/52f0e47311d25da04d101b62).
+
+We decided to build a **simple tool** that would allow anyone with a [Twitter](https://twitter.com) account to *report a parking violation*. All
+you need to do is submit a **geolocated tweet** and a couple of photos
+(**as evidence!**)
+
+Here is an example:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="es" dir="ltr">Ah, ¿no se puede estacionar en paradas de colectivo? Ya fueee... <a href="https://twitter.com/hashtag/InfractoresBA?src=hash">#InfractoresBA</a> <a href="https://t.co/XXmu1pdAib">pic.twitter.com/XXmu1pdAib</a></p>&mdash; E r n e s t o (@_nesto) <a href="https://twitter.com/_nesto/status/722065061309726720">April 18, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+You can check out this tool over here: [http://www.infractoresba.com.ar](http://www.infractoresba.com.ar)
+
+This page shows all the valid parking violations reported by users to
```

Comment:
@etagwerker  what is a "valid" parking violation as opposed to a parking violation? I think you may not need the word valid unless I am missing something.